lib/Constants.js

```diff
@@ -985,7 +985,7 @@ StudyRequestStatus.init({
     color: 'statusAssigned',
     dataViewable: false,
     detailsIndex: 2,
-    editable: false,
+    editable: true,
```

Comment:
Add a comment to indicate that `editable` is now used to indicate whether a request is "closed".  (We used to have a separate `study_requests` column for this, but we don't need that anymore.)